app/javascript/redux/reducers/greetingSlice.js

```diff
@@ -0,0 +1,30 @@
+import { createSlice, createAsyncThunk } from '@reduxjs/toolkit';
+
+const initialState = { message: '' };
+
+export const fetchRandomGreeting = createAsyncThunk(
+  'greeting/fetchRandomGreeting',
+  async () => {
+    const response = await fetch('/random_greeting');
```

Comment:
- [ ] @MNisarAli Kindly check the route used to fetch random greeting , its currently wrong instead of `api/v1/greetings`, its `random_greeting`